src/workingtitle-vcockpits-instruments-airliners/html_ui/Pages/VCockpit/Instruments/Airliners/Shared/WT/AltimeterIndicator.js

```diff
@@ -1760,9 +1760,15 @@ class Jet_PFD_AltimeterIndicator extends HTMLElement {
             //         this.cursorIntegrals[1].update(_altitude, 1000, (hideZeros) ? 1000 : undefined);
             //     }
             // }else{

```

Comment:
Have you tried uncommenting this code and check whether it scrolls?

The `clear`-method is built in and I've had it working with that block of code back in January. The only thing I'm not sure about is whether it scrolled or just appeared.

If it does scroll, there is no need to manipulate the `CJ4_AltitudeScroller` class, in my opinion